test/integration/test/philips_color_test.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+const chakram = require('chakram'), expect = chakram.expect;
+
+var Prepper = require('../lib/testPrepper.js');
+
+function translateHSB(h,s,b) {
+  var transHue = parseInt(65536 * (h % 360 / 360));
+  var transSat = parseInt(254 * s);
+  var transBri = parseInt(254 * b);
+
+  return {'h':transHue,'s':transSat,'b':transBri};
+}
+
+Prepper.makeSuite('Control lights locally',function(){
+
+  var lights;
+  var getterPayload = [{'kind': 'LightOn'}];
+
+  before(Prepper.foxboxManager.foxboxLogin);
+
+  it('check 3 bulbs are registered',function(){
+    
+    // collect all getters for the lightbulbs
+    return chakram.put(Prepper.foxboxManager.getterURL,getterPayload)
+    .then(function(listResponse) {
+      lights = Object.keys(listResponse.body);
+      expect(lights.length).equals(3);
+      expect(listResponse).to.have.status(200);
+    });
+  });
+
+  describe('manipulate lights', function () {
+    before(function() {
+      // turn off all lights in the simulator
+      Prepper.philipshue_server.turnOffLight(1);
+      Prepper.philipshue_server.turnOffLight(2);
+      Prepper.philipshue_server.turnOffLight(3);
+
+      return chakram.put(Prepper.foxboxManager.getterURL,getterPayload)
+      .then(function(listResponse) {
+        lights = Object.keys(listResponse.body);
+        expect(lights.length).equals(3);
+        expect(listResponse).to.have.status(200);
+      });
+    });
+
+    // Currently, there is no mapping between the foxbox
+    // id and the philips hue id until the tag feature is implemented
+    it('Turn on all lights', function(){
```

Comment:
Nit: I don't think we need to have this test since it doesn't check the color state